### ISSUE TITLE
Predictable JXM ports

### ISSUE THREAD

**Issue by GoogleCodeExporter** [mondain/red5#419]
```
Here is a little patch to make JXM ports "stable". It is pure configuration but 
i think it should be "standardized"

diff --git a/src/main/server/conf/red5-common.xml 
b/src/main/server/conf/red5-common.xml
index 37fb492..b99f913 100644
--- a/src/main/server/conf/red5-common.xml
+++ b/src/main/server/conf/red5-common.xml
@@ -31,7 +31,7 @@
     <bean class="org.springframework.jmx.support.ConnectorServerFactoryBean" depends-on="rmiRegistry">
         <property name="objectName" value="org.red5.server:name=rmi" />
         <property name="serviceUrl"
-            
value="service:jmx:rmi://${jmx.rmi.host}/jndi/rmi://${jmx.rmi.host}:${jmx.rmi.po
rt}/red5" />
+            
value="service:jmx:rmi://${jmx.rmi.host}:${jmx.rmi.sport}/jndi/rmi://${jmx.rmi.h
ost}:${jmx.rmi.port}/red5" />
         <property name="environment">
             <!-- the following is only valid when the sun jmx implementation is used -->
             <map>
diff --git a/src/main/server/conf/red5.properties 
b/src/main/server/conf/red5.properties
index 770515e..072cc8b 100644
--- a/src/main/server/conf/red5.properties
+++ b/src/main/server/conf/red5.properties
@@ -91,6 +91,7 @@ proxy.destination_port=1935
 # JMX
 jmx.rmi.host=localhost
 jmx.rmi.port=9999
+jmx.rmi.sport=9998
 jmx.rmi.port.remoteobjects=
 jmx.keystorepass=password
 jmx.mina.monitor.enable=true

Basically changing serviceUrl propery of ConnectorServerFactoryBean (in 
conf/red5-common.xml) to include port after the first hostname, and adding 
property for this in conf/red5.properties.

```

Original issue reported on code.google.com by `st.iv....@gmail.com` on 17 Jul 2013 at 11:41
